appveyor.yml

```diff
@@ -14,15 +14,15 @@ environment:
   - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2022
     TR_ARCH: x86
     QT_VERSION: '5'
-  - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2022
-    TR_ARCH: x86
-    QT_VERSION: '6'
```

Comment:
Please don't remove existing build configurations. Adding the arm64 build doesn't require removing the x86 one.

Reply:
but this matrix is do nothing, right?